### PR TITLE
Hawkular Metrics 0.21.0.Final - Release

### PR DESCRIPTION
**Pull request by stefannegrea**

### CODE REVIEW

File: src/main/jbake/content/blog/2016/10/25/hawkular-metrics-0.21.0.Final-released.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+status=published
+~~~~~~
+
+I am happy to announce release 0.21.0 of Hawkular Metrics.This release is anchored by performance enhancements and general fixes.
+
+Here is a list of major changes:
+
+* Cassandra
+    * **Cassandra 3.0.9 is now the supported version of Cassandra.**
+    * Note: this is a rollback from previously supported version of 3.7 due to Cassandra community recommendations for stability and production deployment. Cassandra 3.7 or 3.9 are still compatible but development and testing now use 3.0.9 release.
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/10/25/hawkular-metrics-0.21.0.Final-released.md:15]: 3.9 -> 3.0.9 ?

File: src/main/jbake/content/blog/2016/10/25/hawkular-metrics-0.21.0.Final-released.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+title=Hawkular Metrics 0.21.0 - Release
+author=Stefan Negrea
+date=2016-10-25
+type=post
+tags=blog, metrics, release
+status=published
+~~~~~~
+
+I am happy to announce release 0.21.0 of Hawkular Metrics.This release is anchored by performance enhancements and general fixes.
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/10/25/hawkular-metrics-0.21.0.Final-released.md:9]: There is also already 21.2 (which we use in h-services) - shouldn' that be announced?

File: src/main/jbake/content/blog/2016/10/25/hawkular-metrics-0.21.0.Final-released.md (hunk 1 above)
Comment (stefannegrea) [reply src/main/jbake/content/blog/2016/10/25/hawkular-metrics-0.21.0.Final-released.md:15]: We rolled back to 3.0.x releases, and 3.0.9 is the latest.

File: src/main/jbake/content/blog/2016/10/25/hawkular-metrics-0.21.0.Final-released.md (hunk 2 above)
Comment (stefannegrea) [reply src/main/jbake/content/blog/2016/10/25/hawkular-metrics-0.21.0.Final-released.md:9]: We do not announce patch release and the patch releases so far only have fixes for OS specific issues. There are no changes to the general distribution.

File: src/main/jbake/content/blog/2016/10/25/hawkular-metrics-0.21.0.Final-released.md (hunk 1 above)
Comment (pilhuhn) [reply src/main/jbake/content/blog/2016/10/25/hawkular-metrics-0.21.0.Final-released.md:15]: " Cassandra 3.7 or 3.9 are still..." - did you really mean 3.9 here?

Comment (stefannegrea) [reply src/main/jbake/content/blog/2016/10/25/hawkular-metrics-0.21.0.Final-released.md:15]: 3.9 works with Hawkular Metrics, we briefly tested it when we had to make the decision to move from 3.7.

**Pull request merged**